tagstudio/src/qt/widgets/preview_panel.py

```diff
@@ -23,6 +23,7 @@
 from src.qt.modals import AddFieldModal
 from src.qt.widgets import (ThumbRenderer, FieldContainer, TagBoxWidget, TextWidget, PanelModal, EditTextBox,
 							EditTextLine, ItemThumb)
+from .video_player import VideoPlayer
```

Comment:
```suggestion
from src.qt.widgets import (ThumbRenderer, FieldContainer, TagBoxWidget, TextWidget, PanelModal, EditTextBox,
							EditTextLine, ItemThumb, VideoPlayer)
```
Should just need to add `VideoPlayer` to the `src.qt.widgets` import on line 24-25

Reply:
Tried that, but python gave me a circular dependency error.

Reply:
Tossed in another comment, I didn't think about import order, `VideoPlayer` needs to be imported in the `__init__` file before `PreviewPanel`